docs/readme-cmake.md

```diff
@@ -129,37 +101,61 @@ You can get a list of integration tests with `ctest --preset <preset>-integratio
 
 Replace `<preset>` with one of the following presets:
 
-- Windows: `clang-cl` (builds with Clang for Windows), `mingw` (builds with MinGW)
-- MacOS: `macos` or `macos-arm` (builds with the default compiler), `xcode` or `xcode-arm` (builds using the XCode toolchain)
+- Windows: `clang-cl` (builds with Clang for Windows), `mingw` (builds with MinGW), `mingw32` (builds with x86 MinGW)
+- MacOS: `macos` or `macos-arm` (builds with the default compiler, for x64 and ARM64 respectively)
 - Linux: `linux` (builds with the default compiler)
 
-# Building with other IDEs
+### Using an IDE
 
-## Building with Visual Studio Code
+Most IDEs have CMake support, and can be used to build the game. We recommend using [Visual Studio Code](#visual-studio-code).
 
-Install the [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools), [CMake Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools), [CMake Test explorer](https://marketplace.visualstudio.com/items?itemName=fredericbonnet.cmake-test-adapter) extensions and open the project folder under File -> Open Folder.
+#### Visual Studio Code
 
-You'll be asked to select a preset. Select the one you want (see the table above). If you get asked to configure the project, click on Yes. You can use the bar at the very bottom to select between different configurations (Debug/Release), build, start the game, and execute the unit tests. On the left you can click on the test icon to run individual integration tests.
+After installing VS Code, install the [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) and [CMake Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools) extensions, and open the project folder under File -> Open Folder.
 
-## Building with Code::Blocks
+You'll be asked to select a preset. Select the one you want (see the list above in the previous section for help). If you get asked to configure the project, click on Yes. You can use the bar at the very bottom to select between different configurations (Debug/Release), build, start the game, and execute the unit tests. On the left you can click on the test icon to run individual integration tests.
 
-If you want to use the Code::Blocks IDE, from the root of the project folder execute:
+#### Visual Studio
+
+We recommend to use [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/#visual-studio-community-2022), because of its better CMake integration. Once you have installed Visual Studio, you can simply open the root folder.
```

Comment:
```suggestion
We recommend using [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/#visual-studio-community-2022) or newer, because of its better CMake integration. Once you have installed Visual Studio, you can simply open the root folder.
```